RUNTIME ERROR: Unexpected type array
-------------------------------------------------
	testdata/std.sort4:1:1-29	builtin function <sort>

std.sort([1, [error "foo"]])

-------------------------------------------------
	During evaluation	


